lib/canvas.js

```diff
@@ -35,6 +36,9 @@ Canvas.prototype = {
     assert(Canvas.supportedFormats.indexOf(format) !== -1,
       '`gmsmith` doesn\'t support exporting a "' + format + '". Please export either a `png` or `jpeg`');
 
+    // Extent to given size
+    canvas.gravity('NorthWest').extent(this.width, this.height, '!');
```

Comment:
I've tried that, it resulted in a non-transparent background for some reason I couldn't determine. Will give it another go tonight when I get some time.